code/datums/diseases/berserker.dm

```diff
@@ -0,0 +1,55 @@
+/datum/disease/berserker
+	name = "Berserker"
+	max_stages = 2
+	stage_prob = 5
+	spread_text = "Non-Contagious"
+	spread_flags = SPECIAL
+	cure_text = "Anti-Psychotics"
+	cures = list("haloperidol")
+	agent = "Jagged Crystals"
+	cure_chance = 10
+	viable_mobtypes = list(/mob/living/carbon/human)
+	desc = "Swearing, shouting, attacking nearby crew members uncontrollably."
+	severity = DANGEROUS
+	disease_flags = CURABLE
+	spread_flags = NON_CONTAGIOUS
+
+/datum/disease/berserker/stage_act()
+	..()
+	if(affected_mob.reagents.has_reagent("thc"))
+		to_chat(affected_mob, "<span class='notice'>You mellow out.</span>")
+		cure()
+		return
+	switch(stage)
+		if(1)
+			if(prob(5))
+				affected_mob.emote(pick("twitch_s", "grumble"))
+			if(prob(5))
+				var/speak = pick("Grr...", "Fuck...", "Fucking...", "Fuck this fucking.. fuck..")
```

Comment:
Lmfao, perfect.

Oh my God, the SNPC chatter never gets old. I'm going to run out of ribs to break every time it puts me into a laughing fit.